### ISSUE TITLE
Request for complete repo project

### ISSUE THREAD

**Issue by sonnyk22** [bbachi/angular-nodejs-docker-compose#2]
Thank you for this great article that you've posted. 

[https://medium.com/bb-tutorials-and-thoughts/angular-local-development-with-docker-compose-13719b998e42](url)

However, it seems that the Github repo that you've mentioned in that article is incomplete, and missing the Node Api project.  

[https://github.com/bbachi/angular-nodejs-docker-compose.git](url)

Is there another repo that you may have that has the complete codebase which includes the `proxy.config.json` as well.

Thank you

**Comment by bbachi** [comment]
Thank you for reading1. I have just updated. Thank you
    On Sunday, May 17, 2020, 12:21:20 PM CDT, Sonny K <notifications@github.com> wrote:  
 
 


Thank you for this great article that you've posted.

https://medium.com/bb-tutorials-and-thoughts/angular-local-development-with-docker-compose-13719b998e42

It seems like the Github repo that you've mentioned in that article is incomplete, and missing the Node Api project.

https://github.com/bbachi/angular-nodejs-docker-compose.git

Is there another repo that you may have that has the complete codebase which includes the proxy.config.json as well.

Thank you

**Comment by sonnyk22** [comment]
Thank you so much for adding those files.

After following both of your great articles and reviewing their source codes, I've created a repo as well based on your code.

[https://medium.com/bb-tutorials-and-thoughts/how-to-serve-angular-application-with-nginx-and-docker-3af45be5b854](url)
[https://medium.com/bb-tutorials-and-thoughts/angular-local-development-with-docker-compose-13719b998e42](url)

My repo uses `Angular`, `Node Api`, and `Nginx`:

[https://github.com/sonnyk22/docker-angular-node-nginx](url)

I've added the needed `Dockerfiles `and `docker-Compose `to my repo as well.  It works great on my `Host `computer, however when I browse to the same containerized web application from another computer I have at home (both running in my home network), the `Angular `and `Nginx `works fine, but the Angular cannot connect to `Node api` container.  I get` ERR_CONNECTION-REFUSED` error as show below.

Have you seen this type of error before?

![err](https://user-images.githubusercontent.com/6744710/82163926-e21a5f80-987b-11ea-8e04-d1db996edb14.png)
